src/Drivers/Chrome.php

```diff
@@ -53,4 +53,9 @@ public function __destruct()
     {
         $this->close();
     }
+
+    protected function runHeadless(): ?string
+    {
+        return config('laravel-console-dusk.headless', false) && ! app()->isProduction() ? null : '--headless';
```

Comment:
Isn't this inversed? So if you set `laravel-console-dusk.headless` to `true`, it will return as non-headless?
```suggestion
        return ! config('laravel-console-dusk.headless', true) && ! app()->isProduction() ? null : '--headless';
```

This will make sure it returns `null` only when `headless` is false. And also defaults to `true` if it's set to null.

Reply:
Totally agree on ya! I had it fixed locally but not pushed yet. Should be fixed now